<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Delete</class>
 <widget class="QMainWindow" name="Delete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ConfirmarBorrarV">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Borrar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="UniqueProp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelarBorrarV">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QLabel" name="TxtID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Propiedad unica</string>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete action</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ConfirmarBorrarV</sender>
   <signal>clicked()</signal>
   <receiver>UniqueProp</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
